packages/shared/dropdown/pvc-dropdown.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from "react";
+import * as _ from 'lodash-es';
+import { PersistentVolumeClaimModel } from "../../models";
+import { pvcResource } from "../resources";
+import {
+  PersistentVolumeClaimKind
+} from "../selectors/k8s";
+import ResourceDropdown from "./ResourceDropdown";
+
+export const PVCDropdown: React.FC<PVCDropdownProps> = (props) => {
+  const { onChange } = props;
+  return (
+    <ResourceDropdown
+      className="nb-endpoints-form-entry__dropdown nb-endpoints-form-entry__dropdown--full-width"
+      resource={pvcResource}
+      resourceModel={PersistentVolumeClaimModel}
+      onSelect={onChange}
+      secondaryTextGenerator={null}
+    />
+  );
+};
+
+export type PVCDropdownProps = {
+  namespace: string;
+  selectedKey: string;
+  onChange: (
+    claimName: string,
+    kindLabel?: string,
+    pvc?: PersistentVolumeClaimKind
+  ) => void;
+  id?: string;
+  desc?: string;
+  dataTest?: string;
+  dataFilter?: (pvc: PersistentVolumeClaimKind) => boolean;
+};
```

Comment:
delete this file... not needed... use ResourceDropDown directly whereever necessary...

Reply:
@SanjalKatiyar as you can see used resource dropdown only. PVC drop-down is just a wrapper.

Reply:
do I still need to remove it?

Reply:
yeah, we don't need it... no need to add a wrapper if we directly can use ResourceDropDown component...
ResourceDropDown is itself a generic component that can handle all types of resources !

Reply:
done removed PVC drop down wrapper and used resource drop-down instead